Digizuite.Test/IntegrationTests/MetadataValueServiceTest.cs

```diff
@@ -31,7 +31,16 @@ public class MetadataValueServiceTest : IntegrationTestBase
 
             var metafields = await fieldService.GetAllMetaFields();
 
-            var labelId = metafields.First(mf => mf.ItemId == fieldItemId).LabelId;
+            int labelId;
+
+            if (metafields.First(mf => mf.ItemId == fieldItemId).Labels.Any())
+            {
+                labelId = metafields.First(mf => mf.ItemId == fieldItemId).Labels.Single().Value.LabelId;
```

Comment:
What happens if a metafield is available in multiple languages? Won't that mean it has multiple labels?

Reply:
Right, there will be multiple languages but since this is a unit test and we have predefined data here, there should not be  problem here I think